dashboard/app/views/levels/_form.html.haml

```diff
@@ -14,13 +14,10 @@
     = render partial: 'levels/editors/all', locals: {f: f} unless @level.is_a? EvaluationMulti
     = render partial: 'levels/editors/md_instructions', locals: {f: f} unless @level.is_a?(DSLDefined) || @level.is_a?(Unplugged)
     = render partial: 'levels/editors/tts', locals: {f: f} unless @level.is_a?(DSLDefined) || @level.is_a?(Unplugged)
-
-    = render partial: 'levels/editors/authored_hints', locals: {f: f} unless @level.is_a?(DSLDefined) || @level.is_a?(Unplugged)
-
     = render partial: 'levels/editors/flappy', locals: {f: f} if @level.is_a? Flappy
     = render partial: 'levels/editors/bounce', locals: {f: f} if @level.is_a? Bounce
     = render partial: 'levels/editors/unplugged', locals: {f: f} if @level.is_a? Unplugged
-    = render partial: 'levels/editors/blockly', locals: {f: f} if @level.is_a? Blockly
+    = render partial: 'levels/editors/blockly', locals: {f: f} if !(@level.uses_droplet?) && @level.is_a?(Blockly)
```

Comment:
it's a little ridiculous that we have so many "Blockly" levels that actually use droplet.

Reply:
Actually, given that, are you sure we don't want to render the `blockly` editor for droplet levels? Particularly if you're moving authored hints into the blockly editor; I don't think we want to lose authored hint for gamelab.

Reply:
Going back through, there are a few things that need to be universally available that are only available in blockly.